backend/citylist.java

```diff
@@ -112,7 +112,16 @@ public void print (String Seperator)
       }
     System.out.print (listofcities.get (listofcities.size () - 1).name);
   }
-  
+  public String send(){
+      String sendstring= new String();
+       for (int i = 0; i < (listofcities.size () - 1); i++)
+      {
+	sendstring= sendstring + listofcities.get (i).name;
```

Comment:
This method sends the string value. This string will be accessed by the frontend to prevent duplicates as well as to display the list